lib/libzfs/libzfs_util.c

```diff
@@ -881,7 +881,7 @@ libzfs_init(void)
 		return (NULL);
 	}
 
-	if ((hdl->libzfs_fd = open(ZFS_DEV, O_RDWR)) < 0) {
+	if ((hdl->libzfs_fd = open(ZFS_DEV, O_RDWR|O_EXCL)) < 0) {
```

Comment:
Why was `O_EXCL` added here?  Could you explain, I thought your comment said it was no longer needed?

Reply:
woops - :-/